### ISSUE TITLE
Can no longer import environments dumped by newman

### ISSUE THREAD

**Issue by glenpike** [a85/POSTMan-Chrome-Extension#4315]
## App Details:
Postman for Mac
Version 6.0.9
OS X 15.6.0 / x64

## Issue Report:
 1. New problem - since 6.0.7 (I think)
 2. It should play nicely with the newman.env file
 3. Console logs: None

When test runs fail in Newman, I sometimes import the newman env which has been dumped
(Newman command is:

```
newman run tools/postman_collection.json --globals tools/globals.postman_globals.json --bail --export-environment /tmp/newman.env
```
The exported newman.env objects are missing the "enabled": true value so a workaround is to add this to each object.

Postman should assume a default value of 'true' for enabled if this is missing rather than bailing on an "unrecognised format" error.

**Comment by kunagpal** [comment]
@glenpike I wasn't able to reproduce this with Newman v3.9.4 and v6.1.4 of the Postman app. Could you share the environment here? Feel free to replace the variable values with dummy equivalents 😄

**Comment by thecuriousdev** [comment]
@glenpike closing this. Feel free to reopen this if you face any further issues.

**Comment by glenpike** [comment]
Hi,

This is still an issue with Newman v3.9.4 and Postman v6.2.3

The newman sample env is below:

```
{
  "id": "ab67cee2-2789-4332-8aec-30bbc75ab9f5",
  "values": [
    {
      "type": "any",
      "value": "3f9056c0-9c79-11e8-92d3-952ffb7f4b5f",
      "key": "userId"
    },
    {
      "type": "any",
      "value": "4018c0a0-9c79-11e8-92d3-952ffb7f4b5f",
      "key": "storeId"
    },
    {
      "type": "any",
      "value": "10",
      "key": "storeNumber"
    },
    {
      "type": "any",
      "value": "40549120-9c79-11e8-92d3-952ffb7f4b5f",
      "key": "storeId2"
    }
  ],
  "_postman_variable_scope": "environment",
  "_postman_exported_at": "2018-08-10T08:42:13.304Z",
  "_postman_exported_using": "Newman/3.9.4"
}
```

If you add a "name" key / value to the root of the JSON it works, so either Newman needs to dump a name, or Postman needs to make that optional.

**Comment by kunagpal** [comment]
@glenpike As can be seen here: https://github.com/postmanlabs/newman/blob/develop/CHANGELOG.yaml#L47, this bug was fixed in Newman v3.10.0. Please update to the latest version of Newman 😄

**Comment by glenpike** [comment]
Nice @kunagpal - that's behaving in 3.10.0 now, thank you 👍
